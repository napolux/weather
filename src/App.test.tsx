// import { render } from '@testing-library/react';
// import App from './App';

test('fake test', () => {
  expect(true).toEqual(true);
});
