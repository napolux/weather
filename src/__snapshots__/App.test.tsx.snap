// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
Array [
  <div
    className="ui container"
  >
    <div
      className="ui menu"
    >
      <div
        className="header item"
        onClick={[Function]}
      >
        Weather
      </div>
      <div
        className="item"
        onClick={[Function]}
      >
        <p>
          A project by 
          <a
            href="https://napolux.com"
          >
            Francesco Napoletano
          </a>
        </p>
      </div>
    </div>
  </div>,
  <div
    className="ui container"
  >
    <div
      className="ui cards"
    />
  </div>,
]
`;
